test/unit/sdam/srv_polling.test.js

```diff
@@ -454,6 +439,33 @@ describe('Mongos SRV Polling', function () {
           new TopologyDescription(TopologyType.Sharded)
         )
       );
+      // This changes the topology to a sharded cluster
+      // We now have topology.s.handleSrvPolling assigned
+      expect(topology.s.handleSrvPolling).to.be.a('function');
+    });
+
+    it('two topologyDescriptionChanged events are emitted when receiving an SRV update with one server', function (done) {
```

Comment:
This is still a bug though, right? Is there a ticket for it?

Reply:
I don't think so, meant to clarify, but its good to have visibility here on the PR. 
The topology emits a change event when it receives the SRVRecord and adds the server to the topology. Another change event will be fired for each server as the SDAM machinery runs discovery on the new host names received. It's my understanding that this intentional.
Maybe the test name is still not great, its not explicitly two events, its 1 event + number of servers in the SRV resolution.

Reply:
updated test looks good! :)